index.html

```diff
@@ -93,3 +93,424 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hesitate to
+          contact me.
+        </p>
+        <p class="AboutMeTextSection-D">
+          Hey there! I can help you build a product, feature or website. Look
+          through some of my work and experience! If you like what you see and
+          have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <div>
+          <button class="ResumeButton ResumeButton-D" type="button">
+            Get My Resume
+          </button>
+        </div>
+        <div class="secondBackgroundImage"></div>
+        <div class="secondBackgroundImage-D"></div>
+        <hr class="AboutLine2" />
+        <div class="MySkills">
+          <ul class="containerTech">
+            <ul class="tech1">
+              <li class="Language1">
+                <img src="Rectangle.png" alt="Rectangle" /> Language
+              </li>
+              <ul class="TechStacks techstacks1D">
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+              </ul>
+            </ul>
+
+            <ul class="tech2">
+              <li class="Language1">
+                <img src="Rectangle2.png" alt="Rectangle" /> Frameworks
+              </li>
+              <ul class="TechStacks">
+                <li>React.js</li>
+                <li>Ruby on Rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+              </ul>
+            </ul>
+
+            <ul class="tech">
+              <li class="Language1">
+                <img src="Rectangle3.png" alt="Rectangle" /> skills
+              </li>
+              <ul class="TechStacks">
+                <li>Database Management</li>
+                <li>Version Control</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Web Developement</li>
+              </ul>
+            </ul>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <!-- html form -->
+
+    <section class="Contact" id="Contact">
+      <div class="form">
+        <a href="#form"></a>
+        <div class="mainContent">
+          <h2 class="smallPara">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+          <form
+            class="firstForm"
+            id="form"
+            method="post"
+            action="https://formspree.io/f/xgedqoye"
+          >
+
+            <ul class="input">
+              <div class="subInput">
+              <li class="newInputName">
+                <input
+                  name="name"
+                  type="text"
+                  id="Name"
+                  placeholder="Full name"
+                  required
+                />
```

Comment:
- [ ] Great job on implementing the input fields! However, as mentioned in the previous review, the height of the input fields needs to be adjusted to match the design. Please refer to the design specifications and update your code accordingly. Using padding can be a suitable approach to achieve the desired height. Keep up the good work! 💪🎉
![image](https://github.com/bhushan354/My-Portfolio/assets/55840999/39f48245-48db-4f10-9c2a-a9a6538a0eb3)